apiml-package/src/main/resources/bin/start.sh

```diff
@@ -207,6 +211,11 @@ else
     externalProtocol="http"
 fi
 
+if [ -n "${ZWE_configs_storage_vsam_name}" ]
+then
+    VSAM_FILE_NAME=//\'${ZWE_configs_storage_vsam_name}\'
+fi
```

Comment:
to try keep a consistent style in the file

```suggestion
if [ -n "${ZWE_configs_storage_vsam_name}" ]; then
    VSAM_FILE_NAME=//\'${ZWE_configs_storage_vsam_name}\'
fi
```